inst/tests/tests.Rraw

```diff
@@ -3959,11 +3959,14 @@ test(1215,
    data.table(N_ID=175931L, VISIT_DATE="2013-03-08T23:40:30", REQ_URL='http://aaa.com/rest/api2.do?api=getSetMobileSession&data={"imei":"60893ZTE-CN13cd","appkey":"android_client","content":"Z0JiRA0qPFtWM3BYVltmcx5MWF9ZS0YLdW1ydXoqPycuJS8idXdlY3R0TGBtU', REQType=2L)
 )
 test(1216.1, fread('A,B,C\n1.2,Foo"Bar,"a"b\"c"d"\nfo"o,bar,"b,az""\n'),
```

Comment:
This looks confusing: do you mean to have 
`'A,B,C\n1.2,Foo"Bar,"a"b"c"d"\nfo"o,bar,"b,az""\n'`
or
`'A,B,C\n1.2,Foo"Bar,"a"b\\"c"d"\nfo"o,bar,"b,az""\n'` ?

Reply:
Good point.  I'm not sure what is meant by that extra \ in that test before the `\"c`.  It's superfluous since opening the string `'` automatically escapes any `"` inside the string.  I checked that as follows : 
```
> identical('A,B,C\n1.2,Foo"Bar,"a"b"c"d"\nfo"o,bar,"b,az""\n',
+           'A,B,C\n1.2,Foo"Bar,"a"b\"c"d"\nfo"o,bar,"b,az""\n')
[1] TRUE
```
So out of the two choices you present, the first one is what it does.  Maybe the second one is what was intended.

I'll add them all in, with a comment alongside.